components/integrations/commands_container/commands_container.tsx

```diff
@@ -1,16 +1,27 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import {Route, Switch, Redirect} from 'react-router-dom';
 
+import {Team} from 'mattermost-redux/types/teams';
+
+import {UserProfile} from 'mattermost-redux/types/users';
+
+import {RelationOneToOne} from 'mattermost-redux/types/utilities';
+
 import InstalledCommands from 'components/integrations/installed_commands';
 import AddCommand from 'components/integrations/add_command';
 import EditCommand from 'components/integrations/edit_command';
 import ConfirmIntegration from 'components/integrations/confirm_integration';
 
-const CommandRoute = ({component: Component, extraProps, ...rest}) => ( //eslint-disable-line react/prop-types
+interface IProps {
+    component: any;
```

Comment:
Can this not be `Component`?

Reply:
This is causing a slew of other breakages. I'm not sure how to fix those. Leaving as is.